README.md

```diff
@@ -1,8 +1,62 @@
+
+**Familiar:** JavaScript, HTML/CSS, Android development, SystemVerilog, Computer Graphics with OpenGL/GLSL, Operating Systems
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_OscarLiang](https://github.com/jarviscanada/jarvis_data_eng_OscarLiang)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OscarLiang/tree/master/linux_sql)]: Developed a tool for monitoring resource usage in a network of Linux computers. Utilized bash scripting for monitoring and a PSQL instance to persist data.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OscarLiang/tree/master/core_java)]:
+      
+  - Twitter App: Developed an application that allows users to post, read, and delete tweets using the Twitter REST API. Programmed in Java with Maven and Spring dependency management and tested application using JUnit and Mockito.
+  - JDBC App: Implemented a simple application to store customer and sale information for a sample company for the purpose of exploring the JDBC library and common data persistence design patterns. Programmed in Java with a PSQL instance for data storage.
+  - Grep App: Programmed an application that achieves similar functionality to a simplified version of the Linux grep command. Developed in Java for the purpose of exploring common Java libraries.
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OscarLiang/tree/master/springboot)]: Developed a stock trading simulator application that uses real stock market data to simulate traders buying and selling stocks. Implemented a RESTful API in Java with Springboot that uses Maven for dependency management, PSQL for data storage, JUnit and Mockito for testing, and makes API requests to IEX Cloud for stock market data.
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OscarLiang/tree/master/python_data_anlytics)]: In development
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OscarLiang/tree/master/hadoop)]: In development
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OscarLiang/tree/master/spark)]: In development
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OscarLiang/tree/master/cloud_devops)]: In development
+
+
+## Highlighted Projects
+**CardMaestro**: Developed website in a team using Agile development practices that allowed users to sign in, create a card collection, and discuss card trades with other users. Implemented in Ruby on Rails with an HTML/CSS and JavaScript frontend.
+
+**Capstone Project - ATM Accessibility Improvement**: Collaborated with a multi-disciplinary engineering team that worked with TD to improve the design of ATMs from an accessibility standpoint. Designed a product that aimed to improve ease of use of ATMs for people with physical and visual impairments.
```

Comment:
did you use any technologies for this work? e.g. software/framework/libraries, design methodologies, documents/reports.

What was the result and impact?  did the user accept or use your design? any feedback/reward?